### PR TITLE
ACP-2601: How to embed Stripe App payment page to your storefront application

### PR DESCRIPTION
**Pull request by matweew**
## PR Description

Ticket: https://spryker.atlassian.net/browse/ACP-2601

## Checklist
- [x] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/project-guidelines-for-stripe.md [hunk 1]
```diff
@@ -148,6 +148,121 @@ In the default OMS configuration, seven days are allocated to Stripe to complete
...
+    <meta charset="UTF-8">
+    <title>Order payment page</title>
+</head>
+<body>
+<iframe src="<?php echo base64_decode($_GET['url']) ?>" width="100%" height="700px" style="border:0"></iframe>
+</body>
+</html>
+```
+
+If your Storefront is based on third-party frontend technologies, follow the documentation of your framework to create a page to render the Stripe payment page using query parameters from the redirect URL provided in the Glue API `POST /checkout` response.
```
Comment (andriitserkovnyi) [inline docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/project-guidelines-for-stripe.md:188]: If I'm using a third-party frontend, do I need to do steps 3-6?

File: docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/project-guidelines-for-stripe.md [hunk 2]
```diff
@@ -148,6 +148,121 @@ In the default OMS configuration, seven days are allocated to Stripe to complete
 
 For instructions on using payment details, like the payment reference, from Stripe, see [Retrieve and use payment details from third-party PSPs](https://docs.spryker.com/docs/pbc/all/payment-service-provider/{{page.version}}/base-shop/retrieve-and-use-payment-details-from-third-party-psps.html)
 
+## Embed the Stripe payment page using iframe
+
+By default, the Stripe App payment flow assumes that the payment page is on another domain. When a user places an order, they're redirected to the Stripe payment page. To improve the user experience, you can embed the Stripe payment page directly into your website as follows:
+
+
+1. Create or update `PaymentConfig.php` (`Pyz\Zed\Payment\PaymentConfig`) on the project level with the following configuration:
```
Comment (andriitserkovnyi) [inline docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/project-guidelines-for-stripe.md:156]: Can it be `Pyz\Zed\Payment\PaymentConfig\PaymentConfig.php`?

Comment (matweew) [reply docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/project-guidelines-for-stripe.md:156]: yes, in this case `src/Pyz/Zed/Payment/PaymentConfig.php`
```suggestion
1. Create or update `src/Pyz/Zed/Payment/PaymentConfig.php` with the following configuration:
```

File: docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/project-guidelines-for-stripe.md (hunk 1 above)
Comment (matweew) [reply docs/pbc/all/payment-service-provider/202404.0/base-shop/third-party-integrations/stripe/project-guidelines-for-stripe.md:188]: no, not needed, it's only for Yves

**Pull request merged**